src/server.js

```diff
@@ -73,248 +73,254 @@ var adminGroupPages = require('./pages/admin_groups')(templates, admin_group)
 // none currently
 
 // Application start
+division.getDivisions().then(divisionNavbars => {
+  console.dir(config, { depth: null })
 
-console.dir(config, { depth: null })
-
-var app = express()
-
-var csrfMiddleware = csurf({
-  cookie: true
-})
+  var app = express()
+  global.locals = {
+    divisionsNavbar: divisionNavbars
+  }
```

Comment:
No globals. Let's look at potentially modifying the `config.templates` object we give to pug-tree. This will also resolve the issue where newly created divisions won't show up until app restart.

Reply:
To expand on that point, we can provide a function to the config object that will allow templates to fetch the divisions on the fly.